tests/test_transpiler.py

```
@@ -106,3 +106,129 @@ def test_pipeline_operator_with_args():
+    assert 'true = True' in parsed_code
+    assert 'false = False' in parsed_code
+    assert 'null = None' in parsed_code
+
+def test_else_if_to_elif():
+    clyp_code = 'if (a) {} else if (b) {}'
+    parsed_code = parse_clyp(clyp_code)
+    assert 'elif (b):' in parsed_code
+
+def test_invalid_var_declaration():
+    clyp_code = 'int = 5'
+    with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)